### PR TITLE
guides: update strapi media example

### PR DESCRIPTION
**Pull request by kendallstrautman**
Re this [community convo](https://community.tinacms.org/t/media-manager-failed-to-load-media-strapi/389/2)

### CODE REVIEW

File: content/guides/nextjs/tina-with-strapi/authenticating-with-strapi.md [hunk 1]
```diff
@@ -70,9 +70,7 @@ First we'll enable or disable Tina based on whether we're in preview mode.
     apis: {
       strapi: new StrapiClient(process.env.STRAPI_URL),
     },
-    media: {
-      store: new StrapiMediaStore(process.env.STRAPI_URL),
-    },
+    media: store: new StrapiMediaStore(process.env.STRAPI_URL),
```
Comment (DirtyF) [inline content/guides/nextjs/tina-with-strapi/authenticating-with-strapi.md:73]:
```suggestion
    media: new StrapiMediaStore(process.env.STRAPI_URL),
```

Comment (DirtyF) [reply content/guides/nextjs/tina-with-strapi/authenticating-with-strapi.md:73]: Why this declaration is different from the other one?

Based on the documentation, it looks like you can just specify `media`: https://tina.io/docs/media/#adding-a-media-store

See:
https://github.com/tinacms/tinacms/blob/f33c4baf5c8eae59b8edc9c2a7c7ef22482dc38f/packages/%40tinacms/core/src/cms.ts#L130-L132

**Review by DirtyF — APPROVED** [review]
Looks like we can omit `media.store` and just set `media`.

File: content/guides/nextjs/tina-with-strapi/authenticating-with-strapi.md (hunk 1 above)
Comment (kendallstrautman) [reply content/guides/nextjs/tina-with-strapi/authenticating-with-strapi.md:73]: This is the preferred API

Comment (kendallstrautman) [reply content/guides/nextjs/tina-with-strapi/authenticating-with-strapi.md:73]: oh woops I see that `store` wasn't removed. thanks Frank.

**Review by DirtyF — APPROVED** [review]

**Pull request merged**